tests/integration/golang/aim/run/delete_test.go

```diff
@@ -76,22 +78,27 @@ func (s *DeleteRunTestSuite) Test_Ok() {
 	}
 	for _, tt := range tests {
 		s.T().Run(tt.name, func(T *testing.T) {
-			originalMinRowNum, originalMaxRowNum, err := s.runFixtures.FindMinMaxRowNums(context.Background(), s.runs[0].ExperimentID)
-			assert.NoError(s.T(), err)
+			originalMinRowNum, originalMaxRowNum, err := s.runFixtures.FindMinMaxRowNums(
+				context.Background(),
+				s.runs[0].ExperimentID,
+			)
+			assert.Nil(s.T(), err)
 
-			var resp any
+			var resp fiber.Map
 			err = s.client.DoDeleteRequest(
 				fmt.Sprintf("/runs/%s", tt.request.RunID),
 				&resp,
 			)
 			assert.Nil(s.T(), err)
 
 			runs, err := s.runFixtures.GetTestRuns(context.Background(), s.runs[0].ExperimentID)
-			assert.NoError(s.T(), err)
+			assert.Nil(s.T(), err)
```

Comment:
why do you change all of the NoError assertions to Nil?

Reply:
because under the hood, NoError do next stuff:
```
func NoError(t TestingT, err error, msgAndArgs ...interface{}) bool {
	if err != nil {
		if h, ok := t.(tHelper); ok {
			h.Helper()
		}
		return Fail(t, fmt.Sprintf("Received unexpected error:\n%+v", err), msgAndArgs...)
	}

	return true
}
```
is it is fully enough to just check for `NotNill`.

Reply:
I think the assertion "NoError" is a better description of what we are asserting

Reply:
probably the same answer. Not a big deal which to use. I've changed it just have the same approach everywhere. We can use `NoError` but in that case let's use it everywhere.